src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -32,6 +33,36 @@
  */
 public class OptionData implements SerializableData
 {
+    /**
+     * The highest positive amount Discord allows the {@link OptionType#NUMBER NUMBER} type to be.
+     */
+    public static final double MAX_POSITIVE_NUMBER = (1L << 53);
```

Comment:
This number is an exclusive upper bound.  The client will fail when using an option that is this value or higher.
![image](https://user-images.githubusercontent.com/5003579/129821484-55357328-2990-4cb3-ae12-543cdf5b45bd.png)
Numbers below this will succeed.
![image](https://user-images.githubusercontent.com/5003579/129821561-369d2cc5-54a5-4015-881e-eec454b63bcf.png)

Reply:
So what now?

Reply:
The constant should define what is the largest _valid_ value.
So ensure that the value itself is valid if it were used

Reply:
Repeating myself:
> So what now? Decrease 53 by one?

To be more precise: Use `(1L << 53) - 1` for positive and `(1L << 53) + 1` for negative value?

Reply:
Yup, which it looks like you've already done 👍